examples/apps/ai-collab/src/types/sharedTreeAppSchema.ts

```diff
@@ -37,7 +37,7 @@ export class SharedTreeTask extends sf.object("Task", {
 	}),
 	status: sf.required(sf.string, {
 		metadata: {
-			description: `The status of the task as either "todo", "in-progress", or "done"`,
+			description: `The case-sensitive status of the task which can ONLY be either one of the following values "To Do", "In Progress", or "Done"`,
```

Comment:
Nit:
The case-sensitive priority of the task can ONLY be one of three levels: "Low," "Medium," or "High."